src/ontology/cl-edit.owl

```diff
@@ -27689,6 +27685,17 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
+# Class: obo:CL_4023043 (meis2 expressing cortical GABAergic cell (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33372656") obo:IAO_0000115 obo:CL_4023043 "A GABAergic type located in the cerebral cortex that expresses the gene Meis2 in mouse")
```

Comment:
Recommend change to "A GABAergic cell located in the cerebral cortex that expresses meis2 in mouse."

Reading very carefully, the placement of 'in mouse' creates a slight ambiguity, but that's quibbling.